sdk/verify.go

```diff
@@ -0,0 +1,90 @@
+// Copyright 2024 Edgeless Systems GmbH
+// SPDX-License-Identifier: AGPL-3.0-only
+
+package sdk
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"log/slog"
+	"net"
+
+	"github.com/edgelesssys/contrast/internal/atls"
+	"github.com/edgelesssys/contrast/internal/grpc/dialer"
+	"github.com/edgelesssys/contrast/internal/manifest"
+	"github.com/edgelesssys/contrast/internal/userapi"
+)
+
+// Client is used to interact with a Contrast deployment.
+type Client struct {
+	log *slog.Logger
+}
+
+// New returns a Client.
+func New(log *slog.Logger) Client {
+	return Client{
+		log: log,
+	}
+}
+
+// Verify checks if a given manifest is the latest manifest in the given history.
+func Verify(expected []byte, history [][]byte) error {
```

Comment:
nit: makes usage clearer
```suggestion
func Verify(expectedManifest []byte, coordinatorManifests [][]byte) error {
```